docs/VLSI/Tutorial.rst

```diff
@@ -78,17 +78,13 @@ Pull the Hammer environment into the shell:
 
 Building the Design
 --------------------
-To elaborate the ``Sha3RocketConfig`` (Rocket Chip w/ the accelerator) and set up all prerequisites for the build system to push just the accelerator + hard macro through the flow:
+To elaborate the ``TinyRocketConfig`` and set up all prerequisites for the build system to push just the accelerator + hard macro through the flow:
 
 .. code-block:: shell
 
-    make buildfile MACROCOMPILER_MODE='--mode synflops' CONFIG=Sha3RocketConfig VLSI_TOP=Sha3AccelwBB
-
-The ``MACROCOMPILER_MODE='--mode synflops'`` is needed because the ASAP7 process does not yet have a memory compiler, so flip-flop arrays are used instead. This will dramatically increase the synthesis runtime if your design has a lot of memory state (e.g. large caches). This change is automatically inferred by the makefile but is included here for completeness.
```

Comment:
So ASAP7 SRAMs work of-out-the-box now?

Reply:
They're still dummy blocks, but they are targetable for behavioral sim, synthesis, and P&R only.